src/Libraries/RevitNodes/GeometryConversion/RevitToProtoSolid.cs

```diff
@@ -28,10 +28,22 @@ public static class RevitToProtoSolid
             {
                 srfs.AddRange(face.ToProtoType(false));
             }
-            var converted = Solid.ByJoinedSurfaces(srfs);
+
+            Solid converted = null;
+            try
+            {
+                converted = Solid.ByJoinedSurfaces(srfs);
+            }
+            catch (Exception e)
```

Comment:
In that case it would be better to just return `null` from the `catch` block. Then you wouldn't have to do the `null` check on `converted` below, you can simply return it at the end of the function.